rspace/src/test/scala/coop/rchain/rspace/HotStoreSpec.scala

```diff
@@ -660,7 +660,7 @@ trait HotStoreSpec[F[_], M[_]] extends FlatSpec with Matchers with GeneratorDriv
                   )
               res   <- hotStore.removeJoin(channel, toRemove).attempt
               cache <- state.read
-              _     <- checkRemoval(res, cache.joins(channel), cachedJoins, index)
+              _     <- checkRemoval(res, cache.joins(channel), cachedJoins, index, shouldFail = false)
```

Comment:
parameters like `shouldFail` make the assertion less readable. I'd rather duplicate the assertions

Reply:
Sure, good idea